ups/drp_pipe.table

```diff
@@ -3,6 +3,15 @@
 # - Common third-party packages can be assumed to be recursively included by
 #   the "base" package.
 setupRequired(base)
+setupRequired(obs_subaru)
+setupRequired(obs_lsst)
+setupRequired(obs_decam)
+setupRequired(obs_subaru)
+setupRequired(pipe_tasks)
+setupRequired(meas_algorithms)
+setupRequired(jointcal)
+setupRequried(fgcmcal)
+setupRequired(ctrl_bps)
```

Comment:
Did we have a discussion as to why this needs to depend on ctrl_bps at all? Presumably the pipeline can run just fine without it -- you just can't process very much data. Can this specific line be setupOptional?

Reply:
Will add comments.

ctrl_bps was included because this package will eventually BPS config files (or includeable pieces of them), so it will have a dependency in a "a change to ctrl_bps [configuration] APIs may affect this package".  That was raised in the RFC but never _really_ got resolved; my preference may have won out only because I exhausted those who disagreed by giving them too much to read (see "Variant A" [here](https://jira.lsstcorp.org/browse/RFC-775?focusedCommentId=274849&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-274849), [here](https://jira.lsstcorp.org/browse/RFC-775?focusedCommentId=276579&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-276579), and in [the proposal on confluence](https://confluence.lsstcorp.org/display/~jbosch/Reorganizing+Top-Level+Packages+and+Pipelines)).

In any case, there is BPS configuration here now, so I'll remove the dependency for now and we can discuss again when that changes.

Reply:
Ended up doing more than just adding comments - the dependency list has been pretty much completely rewritten, to include:
 - the obs packages whose configs these pipelines will load when expanded
 - the packages that define the PipelineTasks referenced here.